applications/nrf_desktop/src/modules/ble_adv.c

```diff
@@ -172,6 +172,15 @@ static int ble_adv_stop(void)
 	return err;
 }
 
+static void conn_find(struct bt_conn *conn, void *data)
+{
+	struct bt_conn **temp_conn = data;
+
+	/* Peripheral can have only one Bluetooth connection. */
+	BUILD_ASSERT(CONFIG_BT_MAX_CONN == 1);
```

Comment:
check conn info and return conn only if indentity matches with the current one

Reply:
done